README.md

```diff
@@ -293,6 +293,10 @@ Currently Vercel Pro Plan is required to be able to Deploy this application with
 
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fcalcom%2Fcal.com&env=DATABASE_URL,NEXT_PUBLIC_WEBAPP_URL,NEXTAUTH_URL,NEXTAUTH_SECRET,CRON_API_KEY,CALENDSO_ENCRYPTION_KEY&envDescription=See%20all%20available%20env%20vars&envLink=https%3A%2F%2Fgithub.com%2Fcalcom%2Fcal.com%2Fblob%2Fmain%2F.env.example&project-name=cal&repo-name=cal.com&build-command=cd%20../..%20%26%26%20yarn%20build&root-directory=apps%2Fweb%2F)
 
+### Render
+
+[![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy?repo=https://github.com/phishy/docker-render)
```

Comment:
should this be /phishy/?

Reply:
we can open a new repo if you want under calcom/docker-render or something

Reply:
Exactly

Reply:
or should we put it in calcom/docker? https://github.com/calcom/docker

Reply:
Actually I think that might work. I'll submit a PR there.

Reply:
cool. let me know and i will archive the new repo I just made